be1-go/hub/organizer.go

```diff
@@ -228,6 +228,8 @@ func (o *organizerHub) createLao(publish message.Publish) error {
 	laoChannelID := "/root/" + encodedID
 
 	laoCh := laoChannel{
+		rollCall{},
+		make(map[string]struct{}),
```

Comment:
Please use the named syntax instead of positional one. Allows readers to know what field this map is for easily without looking up the struct :)